static/js/components/UploadPhotometry.jsx

```diff
@@ -0,0 +1,382 @@
+                  control={control}
+                  rules={{ validate: validateCsvData }}
+                />
+              </FormControl>
+            </Box>
+            <Box m={1} style={{ display: "inline-block" }}>
+              <Box component="span" m={1}>
+                <FormControl className={classes.formControl}>
+                  <InputLabel id="delimiter-label">Delimiter</InputLabel>
+                  <Controller
+                    as={(
+                      <Select labelId="delimiter-label">
+                        <MenuItem value=",">
+                          Comma
+                        </MenuItem>
+                        <MenuItem value={`\\s+`}>
+                          Whitespace
+                        </MenuItem>
+                      </Select>
+                    )}
+                    name="delimiter"
+                    control={control}
+                    rules={{ required: true }}
+                    defaultValue=","
+                  />
+                </FormControl>
+              </Box>
+              <br />
+              <Box component="span" m={1}>
+                {
+                  errors.instrumentID && (
+                    <FormValidationError
+                      message="Select an instrument"
+                    />
+                  )
+                }
+                <FormControl className={classes.formControl}>
+                  <InputLabel id="instrumentSelectLabel">
+                    Instrument
+                  </InputLabel>
+                  <Controller
+                    as={(
+                      <Select labelId="instrumentSelectLabel">
+                        <MenuItem value="multiple" key={0}>
+                          Multiple (requires instrument_id column below)
+                        </MenuItem>
+                        {
+                          instrumentList.map((instrument) => (
+                            <MenuItem value={instrument.id} key={instrument.id}>
+                              {`${instrument.name} (ID: ${instrument.id})`}
+                            </MenuItem>
+                          ))
+                        }
+                      </Select>
+                    )}
+                    name="instrumentID"
+                    rules={{ required: true }}
+                    control={control}
+                    defaultValue=""
+                  />
+                </FormControl>
+              </Box>
+            </Box>
+            <Box m={1}>
+              <HtmlTooltip
+                interactive
+                title={(
+                  <>
+                    <p>
+                      Use this form to upload flux- or mag-space photometry data
+                      (only one type per request, not mixed).
+                    </p>
+                    <p>
+                      Required fields (flux-space):&nbsp;
+                      <code>mjd,flux,fluxerr,zp,magsys,filter[,instrument_id]</code>
```

Comment:
Maybe indicate that instrument id is only required when someone selects Multiple instruments from the drop down on the right hand side. What happens when someone selects `instrument_id = 1` from the drop down and specifies `instrument_id=2` in the CSV upload?

Reply:
Done. If an instrument is selected in the select input, and the user provides an instrument_id column, it shows a validation error message: "Invalid input: instrument_id already specified in select input"

Reply:
(& if "Multitple (provide instrument_id in CSV)" is selected, and they don't provide that column, that also shows a validation error. Likewise if they haven't chosen an option for that select. That covers every case)